bip-0001.mediawiki

```diff
@@ -67,7 +67,7 @@ Each BIP should have the following parts:
 
 * Abstract -- a short (~200 word) description of the technical issue being addressed.
 
-* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example) or licensed under the Open Publication License.
+* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example).
```

Comment:
A quick grep shows that the only BIPs with alternative licenses are:
- bip-0002:  dual-licensed under the Open Publication License and BSD 2-clause license
- bip-0145: dual-licensed under the Open Publication License and BSD 2-clause license

We should just try to get these re-licensed. I don't see anything under CC-BY-SA or CC0. Or I may have missed something?

BTW BIP1 **itself** also needs a copyright heading.